### PR TITLE
Rename Docker image and minor fix for build script

### PR DESCRIPTION
**Pull request by mabelzhang**
- Rename Docker image to `mbari_wec` to match [DockerHub repo](https://hub.docker.com/r/mbari/mbari_wec)
- Minor fix for build script:
  Do not tag with extra suffix (`_nvidia` | `_no_nvidia`) if just building nvidia image.
  Previously, it mistakenly tagged the nvidia base image too, resulting in an extra tag like `nvidia_opengl_ubuntu22_nvidia`.
  The suffix was only meant for the main `buoy` image.
- Add instructions to README for pushing to DockerHub

### CODE REVIEW

File: docker/build.bash [hunk 1]
```diff
@@ -75,6 +75,9 @@ echo "Building $image_name with base image $base"
 docker build --rm -t $image_plus_tag --build-arg base=$base --build-arg user_id=$user_id $DIR/$image_name
 echo "Built $image_plus_tag"
 
-# Extra tag in case you have both the NVIDIA and no-NVIDIA images
-docker tag $image_plus_tag $image_name$image_suffix:latest
-echo "Tagged as $image_name$image_suffix:latest"
+# Don't add extra tage if just building the nvidia image
```
Comment (andermi) [inline docker/build.bash:78]:
```suggestion
# Don't add extra tag if just building the nvidia image
```

Comment (andermi) [inline docker/build.bash:78]:
```suggestion
# Don't add extra tag if just building the NVIDIA image
```

**Comment by mabelzhang** [comment]
Okay, I've tested this. You should be able to follow the README to build with the new Docker image name (`mbari_wec`) and push to DockerHub.

Feel free to try out the instructions and push to DockerHub. We're not going to push a final image anyway until logging is in.

**Review by quarkytale — APPROVED** [review]

**Comment by quarkytale** [comment]
Yet to try pushing to DockerHub but other things work fine

**Comment by mabelzhang** [comment]
Thanks!

**Pull request merged**